### DISCUSSION TITLE
Als Admin will ich die Möglichkeit haben die Stundenpläne erstellen zu lassen, um Zeit zu sparen und nicht noch ein anderes Tool verwenden muss.

### DISCUSSION THREAD

**Discussion by Kronprinz03** [SE-TINF22B2/G2-Schoolify#20] [User Stories]
### Beschreibung:
Der Admin soll eingeben können welche Klasse welche Stunden und Räume braucht und welche Lehrer welche Stunden macht und nach diesen Kriterien Stundenpläne erzeugen.

### Akzeptanz-Kriterien:
- [ ] Stundenpläne werden erstellt und falls möglich alle Stunden einer Klasse erfüllt 
- [ ] Aufgaben lasst wird verteilt auf die Lehrer 
- [ ] Keine Stunden nach 17 Uhr
- [ ] Keine Leerstunden falls möglich 
- [ ] Klassenräume werden nach Fächern zu gewiesen falls nötig (Computerraum / Chemieraum etc.)

Extra-Kriterien:

- [ ] Stunden können manuell verschoben werden von dem Admin

### Issues:
- #121

**Comment by tilwbr** [comment]
- wollen wir das echt implementieren? klingt aufwändig

**Comment by tilwbr** [comment]
@two-Gee

**Reply by two-Gee** [reply]
ja, haben wir geplant

**Reply by Kronprinz03** [reply]
Ich weis jetzt nicht was Sache ist aber wir können die UserStory erstmal auf optional setzten

**Reply by laralngn** [reply]
Das machen wir jetzt nicht mehr oder lassen wir es auf optional?
